src/main/java/randoop/util/ClassFileConstants.java

```diff
@@ -662,31 +673,53 @@ static void longConstant(Long value, ConstantSet cs) {
   public static MultiMap<Class<?>, NonreceiverTerm> toMap(Collection<ConstantSet> constantSets) {
     final MultiMap<Class<?>, NonreceiverTerm> map = new MultiMap<>();
     for (ConstantSet cs : constantSets) {
-      Class<?> clazz;
-      try {
-        clazz = TypeNames.getTypeForName(cs.classname);
-      } catch (ClassNotFoundException | NoClassDefFoundError e) {
-        throw new Error("Class " + cs.classname + " not found on the classpath.");
-      }
-      for (Integer x : cs.ints) {
-        map.add(clazz, new NonreceiverTerm(JavaTypes.INT_TYPE, x));
-      }
-      for (Long x : cs.longs) {
-        map.add(clazz, new NonreceiverTerm(JavaTypes.LONG_TYPE, x));
-      }
-      for (Float x : cs.floats) {
-        map.add(clazz, new NonreceiverTerm(JavaTypes.FLOAT_TYPE, x));
-      }
-      for (Double x : cs.doubles) {
-        map.add(clazz, new NonreceiverTerm(JavaTypes.DOUBLE_TYPE, x));
-      }
-      for (String x : cs.strings) {
-        map.add(clazz, new NonreceiverTerm(JavaTypes.STRING_TYPE, x));
-      }
-      for (Class<?> x : cs.classes) {
-        map.add(clazz, new NonreceiverTerm(JavaTypes.CLASS_TYPE, x));
-      }
+      addToConstantMap(cs, map);
     }
     return map;
   }
+
+  /**
+   * Add the constants in a ConstantSet to the given map.
```

Comment:
Are they added as keys?  As values?  If keys, what are the corresponding values?

Reply:
The comment is fixed: "Add all constant values from the given ConstantSet as NonreceiverTerms to their corresponding class in the given map."